app/Resources/views/parts/blog_post/author.html.twig

```diff
@@ -0,0 +1,16 @@
+<div class="blog-post-author-image">
+    {{ ez_render_field(content, 'profile_picture', {
+        'parameters': {
+            'alias': 'blog_post_author'
+        }
+    }) }}
+</div>
+<div class="blog-post-author-name">
```

Comment:
nitpick: I'd add an extra line between `<div>'s, like in `app/Resources/views/full/blog_post.html.twig`